### DISCUSSION TITLE
Once I close the terminal, Aria2 RPC server disconnect

### DISCUSSION THREAD

**Discussion by silverdragon70** [alexhua/Aria2-for-chrome#121] [Q&A] [answered]
I installed the chrome extension and run the aria2 in cmd 
And the extension shows that it is connected to the rpc server 
But once I close the cmd window , the extension lose connection to aria and be not functioning 
What should I do keep it functioning and connected ,rather than keeping cmd window active 
Is there any additional software to install ?
Note: I also installed AriaNG software as EXE but the problem in the extension and the software

**Comment by alexhua** [comment, marked as answer]
The ariaNg is just the frontend of aria2. Our extension is a bridge between ariaNG and aria2. There will be a solution that let you keep aria2 as a background service soon and you can control it easily from task tray on windows platform. Just wait for our v2.0 final release.

**Reply by silverdragon70** [reply]
Thanks for replay
Hoping next version fixing this coming soon
Thanks for your precious work

**Reply by alexhua** [reply]
We have released a tool. Please refer to https://rec.aria2e.ga/en/posts/3/
